src/prpy/planning/snap.py

```diff
@@ -76,58 +76,6 @@ def PlanToConfiguration(self, robot, goal, **kw_args):
         """
         return self._Snap(robot, goal, **kw_args)
 
-    @ClonedPlanningMethod
-    def PlanToEndEffectorPose(self, robot, goal_pose, **kw_args):
```

Comment:
I am in favor of eventually removing `PlanToEndEffectorPose` from `SnapPlanner`, but I am concerned about breaking backwards compatibility in the process. 

The simplest solution is to implement a `PlanToEndEffectorPose` wrapper function on `SnapPlanner` that simply calls `_PlanToIK` on `IKPlanner`. In the process, you may want to rename `_PlanToIK` and (possibly) make it a `@classmethod`.

This is how I maintained backwards compatibility for CBiRRT's `PlanToEndEffectorOffset` method after introducing the `PlanToEndEffectorOffsetTSRAdapter` class in #311.